scripts/leapp_script.py

```diff
@@ -270,7 +288,7 @@ def install_leapp_pkg_corresponding_to_installed_rhui(rhui_pkgs):
 
 
 def remove_previous_reports():
-    print("Removing previous preupgrade reports at /var/log/leapp/leapp-report.* ...")
+    print("Removing previous leapp reports at /var/log/leapp/leapp-report.* ...")
```

Comment:
Just thinking out loud, does leapp archive the report? maybe in that case we should archive it rather than delete it (it's same in conversion script https://github.com/oamg/convert2rhel-worker-scripts/blob/main/scripts/conversion_script.py#L270)

Reply:
From user perspective I think it is worth to archive it as you mentioned. I will add the functionality :+1:

Reply:
Just find out leapp does it automatically :smile: